internal/dataplane/kong_client.go

```diff
@@ -484,6 +485,10 @@ func (c *KongClient) sendToClient(
 	)
 
 	c.recordResourceFailureEvents(entityErrors, KongConfigurationApplyFailedEventReason)
+	// REVIEW: do we record the event for uploading config to Konnect?
```

Comment:
I'm not sure about this one. @czeslavo WDYT?

Reply:
I'm not sure. One reason to not record such events for Konnect I can see is that if a user would like setup some monitoring based on the these events directly, it won't be programatically easy (without some heuristics like verifying the client URL) to tell if it is an event for a regular gateway or Konnect. 

I'd say that if we'd like to emit something for Konnect, we should rather make that a separate event reason, e.g. `KonnectConfigurationSyncFailed`, but I'm not sure if it's in the scope of this issue.

Reply:
Decided not to include updating Konnect. @czeslavo  please create an issue if we need to add events with different reasons on updating Konnect.